server/lib/chat/chat-service.ts

```diff
@@ -174,6 +170,9 @@ export default class ChatService {
       throw new ChatServiceError(ChatServiceErrorCode.ChannelNotFound, 'Channel not found')
     }
 
+    // NOTE(2Pac): This method can technically return an `undefined`, but that would mean we're
+    // trying to add user to a bunch of non-official channels when they create an account which,
+    // you know... we probably shouldn't do.
     const userChannelEntry = await addUserToChannel(userId, channelId, client)
```

Comment:
If this is the place you are assuming it is defined, this is the place you should add the `!`, not everywhere you try to use the value